examples/one-rate-plan/main.tf

```diff
@@ -0,0 +1,83 @@
+##############################################################################
+# Resource Group
+##############################################################################
+
+module "resource_group" {
+  source = "git::https://github.com/terraform-ibm-modules/terraform-ibm-resource-group.git?ref=v1.0.5"
+  # if an existing resource group is not set (null) create a new one using prefix
+  resource_group_name          = var.resource_group == null ? "${var.prefix}-resource-group" : null
+  existing_resource_group_name = var.resource_group
+}
+
+##############################################################################
+# Get Cloud Account ID
+##############################################################################
+
+data "ibm_iam_account_settings" "iam_account_settings" {
+}
+
+##############################################################################
+# Create CBR Zone
+##############################################################################
+# Create COS instance and Key protect instance.
+# Create COS bucket-1 with:
+# - Encryption
+# - Monitoring
+# - Activity Tracking
+# - One Rate Plan & One Rate Active Bucket Storage Class
+module "cos_bucket" {
```

Comment:
I think this example should be very simple. We probably don't need anything related to the CBR rules or Activity tracker. Maybe you can have a look at https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/cos_bucket#example-usage-4 which creates one rate plan for the COS instances